CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 8.2.0 IN PROGRESS
 
+* ESC - Cancel confirmation popup does not appear on user edit page after entering not unique Username. Refs STRIPESFF-39.
```

Comment:
@zburke It is a very interesting and tricky bug. QA could only reproduce it in the ECS environment. Because the username uniqueness can be validated for the current tenant on username input change. That's why the behavior is reproducible via cross-tenant check only.

Reply:
My concern is that you have described ECS-specific behavior but this is not an ECS-specific fix; it will change the behavior for everybody. 

If we can't reproduce the behavior in a single-tenant setting, we should not change the behavior in a single-tenant setting. Instead, we should look at why cross-tenant username validation isn't correctly triggering the navigation-confirmation dialog.